examples/kafka-hub/hub/modules/config/configurations.bal

```diff
@@ -50,10 +51,7 @@ public configurable int MESSAGE_DELIVERY_COUNT = 3;
 # The message delivery timeout
 public configurable decimal MESSAGE_DELIVERY_TIMEOUT = 10;
 
-# The base URL of IDP
-public configurable string MOSIP_AUTH_BASE_URL = "https://host/";
-
-# The token validation URL of IDP
-public configurable string MOSIP_AUTH_VALIDATE_TOKEN_URL = "https://host/oauth2/token";
+# The Oauth2 authorization related configurations
+public configurable types:OAuth2Config OAUTH2_CONFIG = ?;
```

Comment:
since these aren't constants shall we use camelCase ?

Reply:
For all the configurations we hv used the same naming convention. So for time-being will keep this as it is an address this comment in a different PR.